client/src/components/SocialShare.tsx

```diff
@@ -102,12 +110,31 @@ const SocialSharePortfolioWithoutI18n: React.FC<{
       i18n={i18n}
       location={location}
       url={url}
-      twitterMessage={i18n._(
-        t`The ${buildingCount} buildings that my landlord owns 👀... #WhoOwnsWhat @JustFixNYC`
-      )}
-      emailMessage={i18n._(
-        t`The ${buildingCount} buildings owned by my landlord (via JustFix's Who Owns What tool)`
-      )}
+      twitterMessage={
+        location === "summary-tab"
```

Comment:
I'm noticing there's a number of places where this code says `location === "summary-tab"`... it might make things a bit more concise and also less error-prone (e.g. if one of them is misspelled as "summary-tb" that would be bad) to define a boolean variable called `isSummaryTab` that represents this condition?  Eh, just a thought, but not a big deal.

Reply:
Also, I feel like ideally the actual content of all this would just be passed in as a prop by the summary tab page itself (or defined by a context or something in that page), so that this component was generic and didn't need tons of branching logic--it'd also make more sense to find summary tab-specific content in the summary tab page, rather than here, I think... but anyways, I know a lot of this code is legacy and does this kind of thing all over the place already so it's not a priority.

Reply:
yes, both good points! Yeah I think it would be worth it to refactor